.github/workflows/ci-web.yml

```diff
@@ -63,6 +63,11 @@ jobs:
         working-directory: client
         timeout-minutes: 5
 
+      - name: Check prettier
+        run: npx prettier . --check
+        working-directory: client
+        timeout-minutes: 5
```

Comment:
We have a step `Check lint` that run the command `npm run lint` should `prettier` run at that step ?

Reply:
Does it change anything? If it all the same, we can keep them appart, they're not exactly the same thing.

Reply:
I would rather keep the 2 distinct commands; `npm run lint`, already execute 2 distinct tools, it's enough.
And the roles are not identical:
- ESLint is here to watch syntax rules and check if we apply them.
- prettier is more a re-write tool, that will basically help us to automatically change wrong syntax.